### ISSUE TITLE
Upstairs built-in wardrobe finishings

### ISSUE THREAD

**Issue by iteles** [dwyl/home#148]
One of the bedrooms upstairs has a built-in, 3 door wardrobe. The doors are currently a dull green with gold trim 😍 😄 

This will remain a wardrobe for now - a portion of it might even end up being storage space _if_ required (given we already have #147). But more importantly, this needs to be finalised as the doors were taken off the hinges during the course of construction and a portion of the wardrobe was co-opted for electrical tubing/wiring.

Whilst I'm a big fan of painting these doors a bright, stand-out colour...
https://studiodiy.com/sunburst-bifold-closet-door-makeover/
![image](https://user-images.githubusercontent.com/4185328/95118724-b0c70100-0742-11eb-9f9d-0924e6f3d0b2.png)

We'll need to paint these with primer anyway as we won't be keeping them green!

Any changes we want to make (door locks, inside organisation) can be done at a later date, but for now, I propose we need to:
+ [ ] Tidy up the inside of the wardrobe
  + [ ] Remove all the construction rubble and recycling
  + [ ] Add a false wall (peladur or something with easy access) to cover the wiring in the far left of the closet
  + [ ] Sand down _existing_ shelves in preparation for use
+ [ ] Paint the doors with 2 coats of primer/white
+ [ ] Put doors back on hinges
  + @nelsonic This one is debatable! We could leave one section as open shelving for room use (extra storage) for example

**Comment by nelsonic** [comment]
@iteles I think having (_all_) the doors on the cupboard will make it a lot more **tidy** if multiple people are using it. As for the tasks we could remove the construction-related debris _tomorrow_ so it's less messy. And then we could do the painting and priming pretty quickly if you feel you want to _use_ the wardrobe ahead of December. 💭

**Comment by iteles** [comment]
Yep. I was thinking that it would be an easy thing for Carlos to get done tomorrow #149 and even if all we use it for is a linen cupboard ahead of December, I think it would be incredibly useful to have it. 

Agree with the doors though, you’re right, there’s a propensity for mess if multiple people are using an open shelving system for random things! 👍🏻

**Comment by iteles** [comment]
I spoke to Carlos about this yesterday.

Apparently @nelsonic you'd already agreed with him that you would source a piece of wood to cover the piping?

He was a bit reluctant to paint the doors but it _can_ be done.

I've asked Chico for a quote for new MDF doors because those ones are all cracked (which I'm not too worried about but still) - for the record.

**Comment by nelsonic** [comment]
@iteles replacing the doors will be _considerably_ more expensive than using wood filler to tidy up the cracks. If Fernando is free, he could do the work in less than a day. Then it's just a sanding, priming and painting whichever color you want. 

As for the "piece of wood" to cover the piping. I could measure it and get it cut at Leroy. 
(_please open a **separate** issue for it so that we can keep track of the work, thanks!_)

**Comment by iteles** [comment]
@nelsonic I think I could easily get this done in a week (allowing time for paint to dry) with a little help from guests we have staying over if you reckon wood filler would do the trick on the cracks!

Proposed steps:
+ [ ] Ask Chico to sand down the bottoms of the doors this week sometime (they need to be sanded as with the new flooring, they no longer fit)
  + [ ] Potentially as Chico to find new door hinges of the same size as the existing ones too
+ [ ] Lay down newspaper in the workroom next door & move out everything that shouldn't be exposed to dust
+ [ ] Fill in all the door cracks with wood filler
+ [ ] Sand down the doors
+ [ ] 2 coats of primer (if this is enough)
+ [ ] 2 coats of main paint (I propose a warm yellow as above or yellow and grey (TBD if this would look good with the wooden floor, even if just the top row of doors)
+ [ ] Clear out and clean out inside of the wardrobe
+ [ ] Hang doors back on the wardrobe

@nelsonic What do you think?

**Comment by nelsonic** [comment]
@iteles indeed easily achieved. Would you consider axsking Chico if Fernanda could help out? 🤔

**Comment by iteles** [comment]
Fernando is basically MIA and I'm almost more interested in having him spend his time on doing his actual job first (final coats of paint on all the doors and doorways) than having him embark on this and then not finish his actual task.

But thank you! Just wanted to confirm if this was the way to go 🙌

**Comment by nelsonic** [comment]
Mom (@carolrmc) has good wood/carpentry skills so might be able to help with this when they are here. 💭 
Good plan to keep Fernando focussed on his task. Lame that he's "away". Hope he's not ill. 🤞

**Comment by iteles** [comment]
Something like this?
https://www.leroymerlin.pt/Produtos/Tintas-e-drogaria/Restaurar-e-renovar-madeira/Reparacao-e-manutencao/WPR_REF_14724710
![image](https://user-images.githubusercontent.com/4185328/96369204-87e63900-1150-11eb-98b1-779a12e3b0ff.png)

If so, might try and find a more 'in bulk' supplier because those cracks are _the business_ 😆

**Comment by nelsonic** [comment]
@iteles Yeah, if you're just going to paint over it you can use ordinary filler ("massa de acabamento")
https://www.leroymerlin.pt/Produtos/Tintas-e-drogaria/Preparar-antes-de-pintar/Reparacao-da-parede/WPR_REF_19379773 
![image](https://user-images.githubusercontent.com/194400/96369336-338f8900-1151-11eb-9f9d-5cb6f75a40b6.png)
The **wood filler** is more for ***furniture*** where you want to color-match and re-varnish it.
It certainly won't _hurt_ to use it, but it's basically 10x the price and no major _advantage_.

**Comment by iteles** [comment]
Perfect, thank you!

**Comment by iteles** [comment]
Spoken to Chico, he can come and cut the doors _possibly_ Friday or Monday (26-Oct-2020). 
We can move forward with the filler but not with the painting of the bottom doors if he's delayed. 

I'll check in with him later this week.

**Comment by iteles** [comment]
I have organised with Miguel who can come and start tomorrow or Wednesday. We have tentatively agreed Wednesday so that Chico can come and bring our missing door back. Should be acceptable (not raining) outdoor weather both days.
Diária = €70.

**Comment by iteles** [comment]
I had lately envisioned painting the top half of the cupboard in yellow and the bottom half white, with windy watercolour foliage painted over the top, a more varied version of:
![image](https://user-images.githubusercontent.com/4185328/98712369-1fb8fa80-237e-11eb-98c2-fbd82460a749.png)


For right now, this is unnecessarily complex and we just need to get something done. We will absolutely have plants in the room and that will make up for it. We can always paint the white wall _around_ the cupboard with watercolour foliage  😆 

Onto buying the paint.

## Leroy Merlin
In terms of colours, it looks like all the yellows at LM are from the Luxens brand:
![image](https://user-images.githubusercontent.com/4185328/98713138-21cf8900-237f-11eb-8259-3a20d7e9f096.png)

And these two ugly ones:
![image](https://user-images.githubusercontent.com/4185328/98713213-3744b300-237f-11eb-8cb7-7a48ed54a784.png)



This seems like the closest one but we'd need to see it and have a paint swatch really (or buy this small tester can):
![image](https://user-images.githubusercontent.com/4185328/98713029-fb115280-237e-11eb-9279-ac0eb6c000ad.png)



## CIN
There is a CIN shop up the road from us (by the Asian supermarket) which we could check out tomorrow.

It seems that this `botão de ouro` colour is quite similar to what I was originally thinking, somewhere between the two colours highlighted below really:
https://deco.cin.com/resources/card_pdf/0CCAT%202009%2001.pdf
![Screenshot 2020-11-10 at 17 51 47](https://user-images.githubusercontent.com/4185328/98712118-c8b32580-237d-11eb-8f68-b61fa50e527d.png)

**Comment by iteles** [comment]
I'll head to CIN today and see if I can pick up a sample/palette from them 😊 

In terms of timelines, if we end up outsourcing the rest of the painting here because of the fumes, I would prefer that it's done _next_ week, when we already have folks coming in to do painting (https://github.com/dwyl/home/issues/154#issuecomment-725323408) so that we get all the toxic fumes out of the way at once! I'll coordinate.

**Comment by nelsonic** [comment]
@iteles what did you determine was the color for the doors/frames? (it's currently **`white`**) ⚪ 
Do you want to change that (it's probably 2 days worth of handperson time). ⏳ 
If not, what **`else`** needs to be done for this issue to be considered **`done`**? 💭
